packages/vue/src/components/organisms/SfProductCard/SfProductCard.stories.js

```diff
@@ -83,13 +83,44 @@ storiesOf("Organisms|ProductCard", module)
       alert(label) {
         alert(label);
       },
+      handleSelectedColor(color) {
```

Comment:
Two things here:

1. Maybe small naming convention change would make this method easier to understand (`color` => `selectedColor`, `el` => `color`
2. Currently we do not have many colors however in the future there may be more. Here we are looking for one specific color and we are toggling its value and for anything else we are setting `false`. This is not very efficient. The better approach in my opinion would be to find one (first) element of the colors array and toggle it's value.

```js
      handleSelectedColor(selectedColor) {
        const pickedColorIndex = this.colors.findIndex(color => color.label === selectedColor.label);  // here we are returning first element index that matches the query

        this.colors[pickedColorIndex].selected = !this.colors[pickedColorIndex].selected
      },
```

What do you think?